diff --git a/fortune-mod/datfiles/CMakeLists.txt b/fortune-mod/datfiles/CMakeLists.txt
index 8908435..55cd235 100644
--- a/datfiles/CMakeLists.txt
+++ b/datfiles/CMakeLists.txt
@@ -68,7 +68,7 @@ ADD_CUSTOM_TARGET(
 )
 INSTALL(
     FILES ${_install_unrot_cookies}
-    DESTINATION "${LOCALDIR}"
+    DESTINATION "${COOKIEDIR}"
 )
 
 IF (NOT NO_OFFENSIVE)
diff --git a/fortune-mod/datfiles/off/CMakeLists.txt b/fortune-mod/datfiles/off/CMakeLists.txt
index 8a87044..f6dc2b1 100644
--- a/datfiles/off/CMakeLists.txt
+++ b/datfiles/off/CMakeLists.txt
@@ -54,5 +54,5 @@ ADD_CUSTOM_TARGET(
 )
 INSTALL(
     FILES ${_install_rotated_cookies}
-    DESTINATION "${LOCALODIR}"
+    DESTINATION "${OCOOKIEDIR}"
 )
